local/bin/py/build/actions/pull_and_push_file.py

```diff
@@ -30,13 +31,17 @@ def pull_and_push_file(content, content_dir):
             # remove h1 if exists
             file_content = re.sub(re.compile(r"^#{1}(?!#)(.*)", re.MULTILINE), "", file_content, count=1)
             file_content = TEMPLATE.format(front_matter=front_matter, content=file_content.strip())
-    with open(
-        "{}{}{}".format(
-            content_dir,
-            content["options"]["dest_path"][1:],
-            basename(content["options"]["file_name"]),
-        ),
-        mode="w+",
-        encoding="utf-8",
-    ) as f:
-        f.write(file_content)
+        elif "datadog-agent" in content["repo_name"] and "config_template.yaml" in "".join(content["globs"]):
+            process_agent_config(file_content)
+
+    if "output_content" in content["options"] and content["options"]["output_content"] != False:
+        with open(
+            "{}{}{}".format(
+                content_dir,
+                content["options"]["dest_path"][1:],
```

Comment:
instead of slicing this string can we do something to normalize this?
if someone changes the dest_path to not start with `/` this won't work